### ISSUE TITLE
Difficult to properly resize images.

### ISSUE THREAD

**Issue by WELZ-gh** [NickeManarin/ScreenToGif#341]
If I add in an image (as a watermark or something) and it's bigger than the size, it's very difficult to resize it:

I'm proposing to add in a percentage input field (preferably for both H and W) as well allowing for click + drag even if the image extends over the work area:

![0](https://user-images.githubusercontent.com/35234925/41556890-fa9f93de-7309-11e8-995a-c4799f33a284.png)

Currently, to resize, this is what I have to do:

![ssfsfsfsf](https://user-images.githubusercontent.com/35234925/41556746-a9240b0c-7309-11e8-87de-45076b840d37.gif)

**Comment by NickeManarin** [comment]
Nice, I'll do it. :)

Btw, did you notice that there's a numeric up/down with the text "Scale" under the "Opacity" up/down? You can use that one while you proposed feature is not yet implemented.

But of course, it doesn't accept decimals and you can't control width/height separately yet.

**Comment by WELZ-gh** [comment]
Yep, I did notice that. But as you can see in my gif. It doesn't work (isn't being updated as I resize it manually)

And then also, as you said, the decimal points and W×H.

**Comment by WELZ-gh** [comment]
That was a mistake. I want supposed to close it. Oops.

**Comment by NickeManarin** [comment]
Yes, it ignores the size because it doesn't know how to decide which plane X-Y (width-height) to read.
If that value gets changed, the entire picture gets scaled.
